src/plugins/intel_cpu/src/nodes/def_conv.cpp

```diff
@@ -337,35 +353,89 @@ struct jit_uni_def_conv_kernel_f32 : public jit_uni_def_conv_kernel, public jit_
                             cmp(reg_ic_iter, simd_w);
                             jl(ic_loop_tail, T_NEAR);
 
+                            // check zero markers
+                            uni_vbroadcastss(xmm_v1, dword[aux_reg_sampled_wei + ind_off_ll * jcp_.typesize_sampled_wei]);
+                            uni_vbroadcastss(xmm_v2, dword[aux_reg_sampled_wei + ind_off_hl * jcp_.typesize_sampled_wei]);
+                            uni_vbroadcastss(xmm_v3, dword[aux_reg_sampled_wei + ind_off_lh * jcp_.typesize_sampled_wei]);
+                            uni_vbroadcastss(xmm_v4, dword[aux_reg_sampled_wei + ind_off_hh * jcp_.typesize_sampled_wei]);
+
                             size_t input_buffer_off = (size_t) kh * jcp_.kw * jcp_.ic + kw * jcp_.ic;
 
                             uni_vpmovsxdq(xmm_v1_off, xmm_v1_off);
                             uni_vmovq(reg_tmp_64, xmm_v1_off);
                             imul(reg_tmp_64, reg_tmp_64, jcp_.ic * jcp_.typesize_in);
                             add(reg_tmp_64, aux2_reg_input);
+                            //->
+                            uni_vtestps(xmm_v1, xmm_v1);
+                            jz(nullify_v1);
+                            //<-
                             uni_vmovups(vmm_v1, ptr[reg_tmp_64]);
                             uni_vmulps(vmm_v1, vmm_v1, vmm_w1);
+                            //->
+                            jmp(nullify_v1_end, T_NEAR);
+                            L(nullify_v1);
+                            {
+                                uni_vpxor(vmm_v1, vmm_v1, vmm_v1);
+                            }
+                            L(nullify_v1_end);
+                            //<-
```

Comment:
Can we extract this into a lambda function and use it everywhere?

Reply:
Done via inline function-member to prevent recreating of labmbda object